Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -1,41 +1,80 @@
-# Eye Tracking Examples in MRTK
-This page covers how to get quickly started with using Eye Tracking in MRTK by building on our provided MRTK example package.
-The samples let you experience one of our new magical input capabilities: **Eye Tracking**! 
-The demo include a number of different use cases for seamlessly combining information about what you are looking at with **Voice** and **Hand** input. 
-This enables users to quickly and effortlessly select and move holograms across their view simply by looking at a target and saying _'Select'_ or performing a hand gesture. 
+# Eye tracking examples in MRTK
+This page covers how to get quickly started with using eye tracking in MRTK by building on our provided [MRTK eye tracking example package](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking).
+The samples let you experience one of our new magical input capabilities: **Eye tracking**! 
+The demo includes a number of different use cases ranging from implicit eye-based activations to how to seamlessly combine information about what you are looking at with **voice** and **hand** input. 
+This enables users to quickly and effortlessly select and move holographic content across their view simply by looking at a target and saying _'Select'_ or performing a hand gesture. 
 The demos also include an example for eye-gaze-directed scroll, pan and zoom of text and images on a slate. 
 Finally, an example is provided for recording and visualizing the user's visual attention on a 2D slate.
 
+In the following, we'll go into more detail what each of the different samples in the [eye tracking example package](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking) includes:
 
-## Overview of MRTK Eye Tracking Samples
+![List of eye tracking scenes](../Images/EyeTracking/mrtk_et_list_et_scenes.jpg)
```

Comment:
When looking at the preview, some of these images aren't being loaded (they look to be broken). I think that this is just an artifact of how github previews work, but it may be worth looking at these changes in a local docfx build.

Just to verify that the images and layout are all showing as expected:
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DevDocGuide.html#building-docfx-locally

Basically you can see what it will look like for real once it's on the github.io pages. If you need any help with those steps lemme know.

Reply:
Actually, looks like this seems fine - using the diff page apparently isn't a great indication of accuracy, so resolving this one.

Reply:
Thanks, Will!